### ISSUE TITLE
Multipart/Form-Data

### ISSUE THREAD

**Issue by hirenshah** [webhooksite/webhook.site#141]
How comes multipart/form-data isn't supported in this version?

**Comment by bvwidt** [comment]
Agreed. I cannot find my uploaded file in the self hosted version as in the cloud version it's very easy.

I cannot find which version each is running on?

**Comment by fredsted** [comment]
Webhook.site Cloud is a different product than the OSS version. 

We would welcome a PR for this feature.
